.github/workflows/ci.yaml

```diff
@@ -22,7 +23,10 @@ jobs:
       - run: cp .github/workflows/kernel-${{ matrix.mode }}.config .config
```

Comment:
```suggestion
- if: matrix.outputdir == 'srcdir'
  run: cp .github/workflows/kernel-${{ matrix.mode }}.config .config
- if matrix.outputdir == 'builddir'
  run: mkdir build && cp .github/workflows/kernel-${{ matrix.mode }}.config build/.config
```

Should work like this

Reply:
Same here, I tried the `env` stuff to simplify a bit the repetitions, let's see if it works.

Reply:
It seems it won't run the CI unless I commit the suggestion or dismiss it (!)...

Reply:
I don't think that was the reason. The CI showed me an error in the yaml file.

Reply:
Ah, makes sense. They should really show some indicator here for CI configurations errors (i.e. pre-run), rather than make people go there.